faboptions/src/main/java/com/joaquimley/faboptions/FabOptions.java

```diff
@@ -137,7 +137,7 @@ public void setBackgroundColor(@ColorRes int backgroundColor) {
     }
 
 
-    private void setBackgroundColor(Context context, @ColorInt int backgroundColor) {
+    public void setBackgroundColor(Context context, @ColorInt int backgroundColor) {
```

Comment:
So if we're making this as a public API maybe we should have the method as an Overload?

We're using different parameters so it won't clash.